### ISSUE TITLE
DL1 check fails for some runs related to event jumps

### ISSUE THREAD

**Issue by morcuended** [cta-observatory/cta-lstchain#931]
Command:
```
lstchain_check_dl1 --input-file=/fefs/aswg/data/real/running_analysis/20220302/v0.9/tailcut84/dl1_LST-1.Run07217.0107.h5 --output-dir=/fefs/aswg/data/real/running_analysis/20220302/v0.9/tailcut84 --muons-dir=/fefs/aswg/data/real/running_analysis/20220302/v0.9 --omit-pdf --batch
```

Output:
```python
input files: /fefs/aswg/data/real/running_analysis/20220302/v0.9/tailcut84/dl1_LST-1.Run07217.0107.h5
output directory: /fefs/aswg/data/real/running_analysis/20220302/v0.9/tailcut84
Number of pedestal trigger tags: {'trigger_type': 0, 'ucts_trigger_type': 0}
Opening file /fefs/aswg/data/real/running_analysis/20220302/v0.9/tailcut84/dl1_LST-1.Run07217.0107.h5
   pedestals: 0,  flatfield: 677,  cosmics: 52523
multiprocessing.pool.RemoteTraceback: 
"""
Traceback (most recent call last):
  File "/fefs/aswg/software/conda/envs/osa-v0.9.3/lib/python3.8/multiprocessing/pool.py", line 125, in worker
    result = (True, func(*args, **kwds))
  File "/fefs/aswg/software/conda/envs/osa-v0.9.3/lib/python3.8/multiprocessing/pool.py", line 51, in starmapstar
    return list(itertools.starmap(args[0], args[1]))
  File "/fefs/aswg/software/conda/envs/osa-v0.9.3/lib/python3.8/site-packages/lstchain/datachecks/dl1_checker.py", line 285, in process_dl1_file
    dl1datacheck_flatfield.fill_event_wise_info(subrun_index,
  File "/fefs/aswg/software/conda/envs/osa-v0.9.3/lib/python3.8/site-packages/lstchain/datachecks/containers.py", line 140, in fill_event_wise_info
    first_non_jump = np.where(uj==False)[0][0]
IndexError: index 0 is out of bounds for axis 0 with size 0
"""

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/fefs/aswg/software/conda/envs/osa-v0.9.3/bin/lstchain_check_dl1", line 8, in <module>
    sys.exit(main())
  File "/fefs/aswg/software/conda/envs/osa-v0.9.3/lib/python3.8/site-packages/lstchain/scripts/lstchain_check_dl1.py", line 117, in main
    check_dl1(filenames, args.output_dir, args.max_cores, not args.omit_pdf, args.batch)
  File "/fefs/aswg/software/conda/envs/osa-v0.9.3/lib/python3.8/site-packages/lstchain/datachecks/dl1_checker.py", line 144, in check_dl1
    dl1datacheck = pool.starmap(process_dl1_file, func_args)
  File "/fefs/aswg/software/conda/envs/osa-v0.9.3/lib/python3.8/multiprocessing/pool.py", line 372, in starmap
    return self._map_async(func, iterable, starmapstar, chunksize).get()
  File "/fefs/aswg/software/conda/envs/osa-v0.9.3/lib/python3.8/multiprocessing/pool.py", line 771, in get
    raise self._value
IndexError: index 0 is out of bounds for axis 0 with size 0
```

**Comment by morcuended** [comment]
aha!
```
7217,119,DATA,-1,1646282832000000000,1646282832000000000,90,0,3408599300,run_start
```
No first timestamp available

**Comment by maxnoe** [comment]
UCTS becomes available in event_id 23, and from there, every event is tagged as a jump

**Comment by maxnoe** [comment]
@moralejo Any idea how we should address this? Can we rely on the UCTS info after it became available?


Nevermind: UCTS becomes unavailable after some time again...

**Comment by morcuended** [comment]
The same error happens with run 4740 from 2021-05-13. This time run 4740 has a proper UCTS ref timestamp. However, the dl1 check script does not fail for the previous run 4738 which does not have a reference UCTS timestamp.

```
4738,107,DATA,-1,1620950171000000000,1620950171000000000,90,0,6450505700,run_start
4740,185,DATA,1620950411450505968,1620952585000000000,1620950411450505968,90,0,9119406500,ucts
```

Traceback:
```python
input files: /fefs/aswg/data/real/running_analysis/20210513/v0.9/tailcut84/dl1_LST-1.Run04740.0042.h5
output directory: /fefs/aswg/data/real/running_analysis/20210513/v0.9/tailcut84
Number of pedestal trigger tags: {'trigger_type': 925, 'ucts_trigger_type': 925}
Opening file /fefs/aswg/data/real/running_analysis/20210513/v0.9/tailcut84/dl1_LST-1.Run04740.0042.h5
   pedestals: 925,  flatfield: 927,  cosmics: 51148
multiprocessing.pool.RemoteTraceback: 
"""
Traceback (most recent call last):
[...]
  File "/fefs/aswg/software/conda/envs/osa-v0.9.3/lib/python3.8/site-packages/lstchain/datachecks/dl1_checker.py", line 272, in process_dl1_file
    dl1datacheck_pedestals.fill_event_wise_info(subrun_index,
  File "/fefs/aswg/software/conda/envs/osa-v0.9.3/lib/python3.8/site-packages/lstchain/datachecks/containers.py", line 140, in fill_event_wise_info
    first_non_jump = np.where(uj==False)[0][0]
IndexError: index 0 is out of bounds for axis 0 with size 0
"""
```

**Comment by morcuended** [comment]
Another one for which dl1 check script fails: run 3071 from 2020-11-24

**Comment by moralejo** [comment]
Fixed in #932
